easybuild/easyconfigs/p/PCRE2/PCRE2-10.21-foss-2016a.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'PCRE2'
+version = '10.21'
+
+homepage = 'http://www.pcre.org/'
+description = """
+ The PCRE library is a set of functions that implement regular expression pattern matching using the same syntax
+ and semantics as Perl 5.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
```

Comment:
@RvDijk using `SOURCEFORGE_SOURCE` doesn't work here, because the project is named `pcre` on SourceForge, not `pcre2`

So, you'll need to use this:

```python
source_urls = ['http://download.sourceforge.net/pcre']
```